src/GovUk.Education.ExploreEducationStatistics.Common/Model/TimeIdentifier.cs

```diff
@@ -11,6 +11,18 @@ namespace GovUk.Education.ExploreEducationStatistics.Common.Model
     [SuppressMessage("ReSharper", "UnusedMember.Global")]
     public enum TimeIdentifier
     {
+        [TimeIdentifierMeta("Autumn term", "T1", Term, Academic)]
+        AutumnTerm,
+
+        [TimeIdentifierMeta("Spring term", "T2", Term, Academic)]
+        SpringTerm,
+
+        [TimeIdentifierMeta("Autumn and spring term", "T1T2", Term, Academic)]
+        AutumnSpringTerm,
+
+        [TimeIdentifierMeta("Summer term", "T3", Term, Academic)]
+        SummerTerm,
```

Comment:
Have all existing releases been checked to know which publications we need to invalidate caches for?

This change also means any Publications mixing any of these (including Academic year) now need a different `Publication.LatestPublishedReleaseVersionId` setting.

These three methods will now evaluate the latest release (by reverse chonrological order) differently

https://github.com/dfe-analytical-services/explore-education-statistics/blob/a7c66f00655786bc35086988a8210b1f12c7c798/src/GovUk.Education.ExploreEducationStatistics.Content.Model/Repository/Interfaces/IReleaseVersionRepository.cs#L14-L44

Reply:
Here is a list of all the publication's that use T1/T2/T1T2/T3 time identifier's:
![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/21970718/d765f88c-375d-480f-9e63-c0975d931a6d)

I manually checked these publications on prod, and only `Pupil absence in schools in England` and `Suspensions and permanent exclusions in England` will be affected. All the other publication's have release's with different time identifiers.

Strictly, we should invalidate the `Suspensions...` publication's cache after this, but that will only change the release series ordering on release pages - it won't change the latest release. But I'll still do it.